physical/common.go

```diff
@@ -0,0 +1,10 @@
+package physical
+
+func appendIfMissing(slice []string, i string) []string {
```

Comment:
Rather than do this, can you make this a method in `helper/strutil`? There's an existing method to do the search.